src/module-elasticsuite-core/Api/Index/IndexSettingsInterface.php

```diff
@@ -101,4 +101,20 @@ public function getBatchIndexingSize();
      * @return array
      */
     public function getDynamicIndexSettings($store);
+
+    /**
+     * Returns the time elapsed (in seconds) since its creation after which a non-live index is to be considered ghost.
+     *
+     * @return int
+     */
+    public function getTimeBeforeGhost(): int;
```

Comment:
I'm not seeing this being used anywhere...

Reply:
Sorry, I forgot to remove it. 
I was hesitating between two solutions: injecting IndexSetting and IndexSettingHelper in \Smile\ElasticsuiteIndices\Model\IndexStatusProvider, or adding this method to IndexSetting so that only it needs to be injected in IndexStatusProvider. What do you think?

Reply:
So this getTimeBeforeGhost in the IndexSettings model should not be needed for now.